vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/options/options.go

```diff
@@ -135,6 +137,8 @@ func NewCMServer() *CMServer {
 
 // AddFlags adds flags for a specific CMServer to the specified FlagSet
 func (s *CMServer) AddFlags(fs *pflag.FlagSet, allControllers []string, disabledByDefaultControllers []string) {
+	fs.StringVar(&s.OpenShiftConfig, "openshift-config", s.OpenShiftConfig, "indicates that this process should be compatible with openshift start master")
```

Comment:
We might want to make this a hidden flag

Reply:
> We might want to make this a hidden flag

Ok.